.github/workflows/code-validation.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
         go-version-file: 'go.mod'
         cache: true
 
-    - name: Setup Helm (for validate-helm test)
-      if: matrix.test == 'validate-helm'
+    - name: Setup Helm (for validate-helm and precommit tests)
+      if: matrix.test == 'validate-helm' || matrix.test == 'precommit'
```

Comment:
This step takes one second to run 🤷 
```suggestion
```

And I think that this action might use the github actions cache so should be stable to leave it.

Reply:
Thanks, I took your suggestion spercially cuz helm is stable download
https://github.com/scality/mountpoint-s3-csi-driver/commit/babba71b5ddd841067199ab6c981bf58a80fe42e